crates/cast/src/cmd/run.rs

```diff
@@ -238,19 +246,6 @@ impl RunArgs {
 
                     configure_tempo_tx_req_env(&mut env, tx)?;
 
-                    // System transactions may have zero or invalid gas limits from the RPC.
-                    // Override with a reasonable value to allow execution.
-                    if is_known_system_sender(tx.from())
-                        || tx.transaction_type() == Some(SYSTEM_TRANSACTION_TYPE)
-                    {
-                        env.evm_env.cfg_env.disable_block_gas_limit = true;
-                        env.evm_env.cfg_env.disable_balance_check = true;
-                        env.evm_env.cfg_env.disable_nonce_check = true;
-                        env.evm_env.cfg_env.disable_fee_charge = true;
-                        env.evm_env.cfg_env.disable_base_fee = true;
-                        env.tx.gas_limit = u64::MAX;
-                    }
```

Comment:
do we not need this anymore? this was added to be able to replay any transaction in a block in Tempo without `--quick` because in order to replay the block correctly we need to replay the system transactions.